.env.example

```diff
@@ -0,0 +1,10 @@
+MNEMONIC=
+NETWORK=testnet
+PACKAGE_ID=0x91ceca50cfe993ec5575cc7805502ceac2b0724709f0f20589d6f1fe8ec09713
+START_MS=1755621276000
+# 1 days
+DURATION_MS=86400000
+AUCTION_SIZE=1000
+ADMIN_CAP_ID=
+AUCTION_ID=
+CLEAR_PRICE=
```

Comment:
I think we should have all this params directly in TS script, and we can modify it if needed. Only Mnemonic should be env.